config/locales/en.yml

```diff
@@ -433,6 +433,9 @@ en:
     duration: Course Duration
     edit_count: Edit Count
     edit_course_dates: Edit Course Dates
+    embed_course_stats: Embed Course Stats
+    embed_course_stats_heading: To Embed Course Stats
```

Comment:
Maybe instead of a header and then more copy, just the full explanation without a header would be clearer:

`To embed the stats for this program in another website, copy and paste this code snippet:`